src/components/PokemonDetails/PokemonDetails.css

```diff
@@ -0,0 +1,82 @@
+.pokemon-details-page {
```

Comment:
Thanks for adding in the css I wrote! Definitely needs refactored, but I think it's a great start!